core/src/main/scala/org/apache/spark/sql/execution/sources/StoreDataSourceStrategy.scala

```diff
@@ -72,6 +72,20 @@ private[sql] object StoreDataSourceStrategy extends Strategy {
           0,
           Nil,
           (a, f) => t.buildUnsafeScan(a.map(_.name).toArray, f.toArray)) :: Nil
+      case LogicalRelation(_, _, _) => {
+        var foundParamLiteral = false
+        val tp = plan.transformAllExpressions {
```

Comment:
Can you add a comment here on why this has been added (like in https://github.com/SnappyDataInc/snappydata/pull/1035/files#diff-d32d8c68c3b9a5c7e05772e506b0ace5)